src/_check.sh

```diff
@@ -120,3 +121,29 @@ function _check_base_required_packages() {
   fi
   return $status
 }
+
+function check_etl_status() {
+  title "Checking Couchbase-to-Postgres ETL status"
+
+  RAW_OUTPUT=$(compose_client exec plextracapi npm run pg:etl:status)
+
+  # Find the json content by looking for the first line that starts
+  # with an opening brace and the first line that starts with a closing brace.
+  JSON_OUTPUT=$(echo "$RAW_OUTPUT" | sed '/^{/,/^}/!d')
+
+  # Find the summary content by finding the first line that starts
+  # with a closing brace and selecting all remaining lines after that one.
+  SUMMARY_OUTPUT=$(echo "$RAW_OUTPUT" | sed '1,/^}/d')
+
+  ETLS_COMBINED_STATUS=$(echo $JSON_OUTPUT | jq -r .etlsCombinedStatus)
+
+  if [[ "$ETLS_COMBINED_STATUS" == "HEALTHY" ]] 
+  then
+    info "All ETLs are in a healthy status."
+  else
+    error "One or more ETLs are in an unhealthy status."
+  fi
+
+  msg "$SUMMARY_OUTPUT\n"
```

Comment:
I remembered this looking over a different PR, msg should be log for correct formatting.
```suggestion
  log "$SUMMARY_OUTPUT\n"
```